timescaledb/getting-started/query-data.md

```diff
@@ -1,153 +1,135 @@
-# 5. Query your data
+# Query your data
 
 With TimescaleDB, there's no need to learn a custom query language. **TimescaleDB
-supports full SQL**. This means you can put your SQL knowledge to good use and
-use the rich ecosystem of PostgreSQL tools you know and love.
-
-For example, here's how to find the average temperature for each city in the past 2 years:
-
-```sql
---------------------------------
--- Average temperature per city
--- in past 2 years
---------------------------------
-SELECT city_name, avg(temp_c)
-FROM weather_metrics
-WHERE time > now() - INTERVAL '2 years'
-GROUP BY city_name;
-```
-
-And here's how to find the total snowfall for each city in the past 5 years:
-
-```sql
---------------------------------
--- Total snowfall per city
--- in past 5 years
---------------------------------
-SELECT city_name, sum(snow_1h_mm)
-FROM weather_metrics
-WHERE time > now() - INTERVAL '5 years'
-GROUP BY city_name;
-```
-
-
-<highlight type="tip">
-Fun fact: TimescaleDB adds important enhancements to the PostgreSQL query planner
-that improve query reusability for INTERVAL predicates, something PostgreSQL does
-not have.
-</highlight>
-
-## Advanced SQL functions for time-series data
+supports full SQL**. You can use your SQL knowledge along with the rich 
+ecosystem of PostgreSQL tools, and add the extra features and performance of
+TimescaleDB.
+
+
+Here are some query examples so you can get familiar with using SQL alongside
+popular TimescaleDB functions.
+
+## Basic SQL queries
+### Select all stock data from the last day.
+
+   This query uses the [`WHERE`][clause-expressions] clause to only show results for
+   timestamps within the last day. 
+
+   ```sql
+   SELECT * FROM stocks_real_time srt
+   WHERE time > now() - INTERVAL '1 day';
+   ```
+
+### Select the top 10 stock values by price.
```

Comment:
```suggestion
### Select the top 10 stock trades by price.
```

Reply:
These will likely all be the same stock symbol, with similar timeframe. Not sure what we would add to this to make it a little more interesting (I realize these are basic queries). Hmmmm.... 🤔